### PR TITLE
Add support for placeholders

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

**Comment by hedvigoscar** [comment]
This PR implements _most_ of the functionality from Lokalise placeholders.

Notable features that are missing:

- Type-checking on the placeholder (everything must be a string) - Lokalise placeholders support Strings, Ints and Floats
- Float Decimal places

File: src/main/kotlin/com/hedvig/lokalise/repository/LokaliseRepository.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package com.hedvig.lokalise.repository
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.hedvig.lokalise.client.model.LokaliseResponse
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import java.util.Locale
+
+class LokaliseRepository(
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/lokalise/repository/LokaliseRepository.kt:9]: This feels more like the Client now while `LokaliseClient` feels lite the Repository I would switch the name on them or call this one InnerClient

File: build.gradle.kts [hunk 2]
```diff
@@ -20,6 +23,10 @@ dependencies {
     api("com.fasterxml.jackson.core:jackson-databind:2.11.0")
     api("com.fasterxml.jackson.core:jackson-annotations:2.11.0")
     implementation("org.slf4j:slf4j-api:1.7.30")
+
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.1.0")
```
Comment (FredrikAreschoug) [inline build.gradle.kts:27]: :heart:

File: src/main/kotlin/com/hedvig/lokalise/repository/LokaliseRepository.kt (hunk 1 above)
Comment (hedvigoscar) [reply src/main/kotlin/com/hedvig/lokalise/repository/LokaliseRepository.kt]: Good point. I've switched them and attempted to maintain backwards compatibility.

File: src/test/kotlin/com/hedvig/lokalise/client/LokaliseRepositoryTest.kt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import com.hedvig.lokalise.repository.LokaliseRepository
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import pl.miensol.shouldko.shouldEqual
+import java.util.Locale
+
+internal class LokaliseRepositoryTest {
+    @Test
+    fun `should handle placeholders correctly`() {
```
Comment (FredrikAreschoug) [inline src/test/kotlin/com/hedvig/lokalise/client/LokaliseRepositoryTest.kt:13]: Nice with a Test! Would be nice to test multiple translations as well just so we know that works fine

**Pull request merged**